qualtran/bloqs/basic_gates/rotation_test.py

```diff
@@ -20,13 +20,28 @@
 from qualtran._infra.gate_with_registers import get_named_qubits
 from qualtran.bloqs.basic_gates import CZPowGate, Rx, Ry, Rz, XPowGate, YPowGate, ZPowGate
 from qualtran.bloqs.basic_gates.rotation import _rx, _ry, _rz
+from qualtran.resource_counting import GateCounts, get_cost_value, QECGatesCost
+from qualtran.resource_counting.classify_bloqs import bloq_is_rotation, bloq_is_t_like
 
 
 def test_rotation_gates():
```

Comment:
rename this to `test_rotation_with_t_angle` or sth?

Reply:
done